bot/resources/tags/docstring.md

```diff
@@ -0,0 +1,16 @@
+A [`docstring`](https://docs.python.org/3/glossary.html#term-docstring) is a string with triple quotes that often used in file, classes, functions, etc. Docstrings usually has clear explanation, parameter(s) and return type.
+
+Here's an example of usage of a docstring:
+```py
+def greet(name, age) -> str:
+  """
+  :param name: The name to greet.
+  :type name: str
+  :param age: The age to display.
+  :type age: int
+  :return: String of the greeting.
+  """
```

Comment:
You should include an explanation of this function in the example, to show best practice